authors/apps/authentication/views.py

```diff
@@ -72,3 +85,78 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ResetPasswordAPIView(generics.CreateAPIView):
+    """Sends password reset link to email"""
+    serializer_class = ResetSerializer
+
+    def post(self, request):
+
+        username = request.data
+        email = username['email']
+        if email == "":
+            return Response({"errors": {
+                "email": ["An email is required"]}})
+        user = models.User.objects.filter(email=email)
+        if user:
+            token = jwt.encode({"email": email, "iat": datetime.now(),
+                                "exp": datetime.utcnow() + timedelta(minutes=5)},
+                               settings.SECRET_KEY, algorithm='HS256').decode()
+            to_email = [email]
+            subject = "You requested a password Reset"
+            DEFAULT_FROM_EMAIL = "django.unchained500@gmail.com"
```

Comment:
I think the DEFAULT_FROM_EMAIL value should be saved in the path to mask your identity

Reply:
Let me change that